File: 03/14_imports.java
import java.math.*;

public class SuperMain {
    public boolean variable = true;

    public static void main(int x, int y) {
        double res = Math.log(x);
        double res2 = Math.sin(x);
        double res3 = Math.cos(x);
        double res4 = Math.tan(y);
        double res5 = Math.sqrt(y);
    }
}

public class Main extends SuperMain {
    public void mainn(int x, int y) {
        super.main(x, y);
        variable = false;
    }
}

File: code/14_imports.py
import math
class SuperMain:
    variable = True
    
    @staticmethod
    def main(x: int, y: int):
        res = math.log(x)
        res2 = math.sin(x)
        res3 = math.cos(x)
        res4 = math.tan(y)
        res5 = math.sqrt(y)
class Main(SuperMain):
    
    def mainn(self, x: int, y: int):
        super().main(x, y)
        self.variable = False